bin/node/runtime/src/lib.rs

```diff
@@ -1520,11 +1521,13 @@ parameter_types! {
 }
 
 impl pallet_dex::Config for Runtime {
-	type Fee = ConstU64<3>;
+	type Fee = ConstU32<3>;
 	type RuntimeEvent = RuntimeEvent;
 	type Currency = Balances;
 	type AssetBalance = <Self as pallet_balances::Config>::Balance;
+	type PromotedBalance = u128;
```

Comment:
Promoted balance should be higher to the main balance or assets balance

Reply:
current blocker is that U256 doesn't impl IntegerSquareRoot.

Reply:
As soon as there's a new release of primitive-types we can switch this to U256: https://github.com/paritytech/parity-common/issues/717

Reply:
That's published now so have switched it to U256.